doc/source/raysgd/raysgd_pytorch.rst

```diff
@@ -154,17 +143,19 @@ You can customize the exact function that is called by using a customized traini
 Shutting down training
 ----------------------
 
-After training, you may want to reappropriate the Ray cluster. To release Ray resources obtained by the trainer:
+After training, you may want to reappropriate the Ray cluster. To release Ray resources obtained by the Trainer:
 
 .. code-block:: python
 
     trainer.shutdown()
 
-.. note:: Be sure to call ``save`` or ``get_model`` before shutting down.
+.. note:: Be sure to call ``trainer.save()`` or ``trainer.get_model()`` before shutting down.
```

Comment:
Maybe just put this in the code block to make sure people don't miss it?

Reply:
Think the way it is right now (in a highlighted block of text), it'll be OK because there is a specific section on saving/getting the model.